fjord/feedback/tests/test_models.py

```diff
@@ -322,6 +324,8 @@ def test_purge(self):
         eq_(ResponseContext.objects.filter(
             opinion__created__gte=cutoff).count(),
             5)
+        eq_(pre_journal_count+1,
```

Comment:
Nit: There should be a spaces between things. This should be `eq_(pre_journal_count + 1,`.